optimum/exporters/neuron/convert.py

```diff
@@ -713,6 +706,7 @@ def trace_neuronx(
             neuronx_distributed.trace.parallel_model_save(neuron_model, output)
     else:
         # Case 3: Using `torch_neuronx.trace`
+        cpu_backend = get_available_cores() == 0
```

Comment:
I think you can just do
```suggestion
        cpu_backend = (get_neuron_major() == -1)
```
It is going to be faster.